exchanges/btcmarkets/btcmarkets_wrapper.go

```diff
@@ -1091,3 +1099,33 @@ func (b *BTCMarkets) GetHistoricCandlesExtended(ctx context.Context, p currency.
 func (b *BTCMarkets) GetServerTime(ctx context.Context, _ asset.Item) (time.Time, error) {
 	return b.GetCurrentServerTime(ctx)
 }
+
+// UpdateOrderExecutionLimits sets exchange executions for a required asset type
+func (b *BTCMarkets) UpdateOrderExecutionLimits(ctx context.Context, a asset.Item) error {
+	if !a.IsValid() || a != asset.Spot {
```

Comment:
If only spot is supported, is there a need for this double-check?

Reply:
nope, good pick up!